assets/app/views/project.html

```diff
@@ -2,7 +2,7 @@
   <project-page>
     <div ng-controller="OverviewController">
       <tasks></tasks>
-      <h1>Project {{project.displayName || project.metadata.name}}</h1>
+      <h1>Project {{project | displayName || project.metadata.name}}</h1>
```

Comment:
make sure this is actually working correctly, angular syntax can be funny so make sure you dont need to wrap `project | displayName` in parentheses like `(project | displayName) || project.medata.name`